src/views/index.html

```diff
@@ -2,11 +2,70 @@
 
 {% block content %}
 {% if user %}
-<p>
-  Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor voluptates sapiente quidem, tempora aspernatur
-  placeat aliquam soluta fugiat officiis consectetur nihil exercitationem facere laudantium magni iste eum, maiores,
-  voluptate accusamus
-</p>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-third">
+    <form action="/sitedata" method="post" class="form">
+      {{ govukInput({
```

Comment:
Can you put `<input type="hidden" name="_csrf" value="{{ csrf }}" />` in here now that this branch knows about `csrf` 🤓

Reply:
not a `<meta>` tag?